lib/glueby/contract/timestamp.rb

```diff
@@ -70,9 +72,23 @@ def initialize(
       def save!
         raise Glueby::Contract::Errors::TxAlreadyBroadcasted if @txid
 
-        @tx = create_tx(@wallet, @prefix, Tapyrus.sha256(@content), @fee_provider)
+        content = digest_content(@digest)
+        @tx = create_tx(@wallet, @prefix, content, @fee_provider)
         @txid = @wallet.internal_wallet.broadcast(@tx)
       end
+
+      private
+
+      def digest_content(digest = '')
```

Comment:
Since it is initialized in the constructor, you probably don't need the default values for the arguments.

Reply:
That's right. I will fix it.

I have a question.
What do it if digest is gave not supported symbol?

Reply:
It should raise error. Errors should be defined in `lib/glueby/contract/errors.rb`.

Reply:
OK, Thank you.